File: C/IP_C_01/IMC.c
#include <stdio.h>

int main(){
    double weight, height, imc;

    scanf("%lf", &weight);
    scanf("%lf", &height);

    imc = (weight / (height * height));
    if(imc < 18.5) printf("Abaixo do peso\n");
    else if( imc >= 18.5 && imc < 25) printf("Peso normal\n");
    else if( imc >= 25 && imc < 30) printf("Sobrepeso\n");
    else if( imc >= 30) printf("Obesidade\n");

    return 0;
}

File: Java/TrainigAndActivitys/src/IMC.java
public class IMC extends Main{

    public static void IMC(){
        double Altura, Peso, IMC;

        System.out.printf("Digite o seu peso: ");
        Peso = scanner.nextFloat();

        System.out.printf("Digite a sua altura: ");
        Altura = scanner.nextDouble();

        IMC = Peso / (Altura*Altura);

        if(IMC < 18.5) System.out.println("Abaixo do peso!");
        else if(IMC >= 18.5 && IMC < 25) System.out.println("Peso normal!");
        else if(IMC >= 25 && IMC < 30) System.out.println("Acima do peso!");
        else System.out.println("Obesidade!");
    }
}
